docs/development.md

```diff
@@ -6,32 +6,21 @@ This document will explain how to set up Wikijump on your machine for local deve
+The recommended way to install Wikijump is via Docker. Docker is a way of containerizing, or in the case of Windows or Mac, also virtualizing Linux images. It lets you easily create and destroy different Wikijump builds, and it also acts like a sandbox to protect the rest of your system from dependency pollution. 
 
 > ### For Windows:
 >
-> For Windows, you will need WSL2, a way of running a Linux distribution simultaneously with Windows. You will need Windows 10 for this. The only alternative to WSL2 would be using a Linux virtual machine such as [VirtualBox](https://www.virtualbox.org/) - but using a VM isn't recommended and it won't be explained how to use one here.
+> For Windows, you will need WSL2, a way of running a Linux distribution simultaneously with Windows. You will need Windows 10 or 11 for this. The only alternative to WSL2 would be using a Linux virtual machine such as [VirtualBox](https://www.virtualbox.org/) — but using a VM isn't recommended, and is not covered by this document.
 >
 > It is recommended that you use Ubuntu for the WSL2 distribution. Ubuntu in particular has considerations made for WSL2 use and in general will be the most reliable way forward.
 >
-> [WSL2 download and installation page](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+> [WSL2 download and installation page](https://learn.microsoft.com/en-us/windows/wsl/install)
 
 You will need [Docker](https://www.docker.com/) installed and running:
 
-<table>
-  <thead><tr><th>Ubuntu / Ubuntu VM</th><th>Windows via WSL2</th></tr></thead>
-  <tbody valign="top"><tr>
-    <td>
-      <p><pre># apt install docker.io</pre></p>
-      <p>Start the Docker daemon via systemd or whatever your service manager is.</p>
-      <p><pre>$ sudo systemctl enable --now docker.service</pre></p>
-    </td>
-    <td>
-      <p>Install <a href="https://docs.docker.com/docker-for-windows/install-windows-home">Docker Desktop</a>, which you'll be using as the Docker daemon, and leave it running in the background.</p>
-      <p>If you have set <code>appendWindowsPath=false</code> in your WSL config, then you may hit an error along the lines of <code>"docker-credential-desktop.exe": executable file not found in $PATH</code>. In this case you should either add <code>/mnt/c/Program\ Files/Docker/Docker/resources/bin</code> to your PATH, or <a href="https://github.com/rossjrw/dotfiles/blob/3c5445abb138b735cc3caf61f070c9125fa87d2f/.profile#L28">create a symlink</a> in a directory that is in your PATH already.</p>
-    </td>
-  </tr></tbody>
-</table>
+<pre>$ sudo apt install docker.io</pre>
+Then start the Docker daemon via dockerd:</p>
+<pre>$ sudo dockerd</pre></p>
```

Comment:
We should be recommending they start the docker service, not running the daemon manually. I remember talking about a table or similar which showed systemd vs WSL2 `service` invocations for starting the Docker daemon.

Presently this assumes the user is using Ubuntu (or specifically WSL2?), this should get table-ized or put into the "WIndows" quote, or something like that.